src/collectd.conf.pod

```diff
@@ -8300,6 +8300,11 @@ Synopsis:
      CollectVCL         false
      CollectVSM         false
      CollectWorkers     false
+     CollectLCK         false
```

Comment:
Each config option needs a more detailed entry below. It's easiest do copy and adapt the entry for `CollectWorkers`.